packages/bpk-component-input/src/BpkInput-test.js

```diff
@@ -91,12 +91,7 @@ describe('BpkInput', () => {
     jest.spyOn(console, 'error').mockImplementation(() => jest.fn());
     const tree = renderer
       .create(
-        <BpkInput
-          id="test"
-          name="test"
-          value=""
-          clearButtonMode="whileEditing"
-        />,
+        <BpkInput id="test" name="test" value="" clearButtonMode="always" />,
```

Comment:
Not related to the PR, but I noticed `clearButtonMode` is wrong according to the test name 😅

Reply:
Good catch!